doc/README.md

```diff
@@ -1,10 +1,33 @@
 # SUNDIALS Documentation
 
 The SUNDIALS documentation is written using reStructuredText and
-[Sphinx](https://www.sphinx-doc.org/).
+[Sphinx](https://www.sphinx-doc.org/). We host the generated HTML documentation
+at https://sundials.readthedocs.io.
 
 To build the documentation with Sphinx you will need Python 3.9+. Sphinx and the
-necessary extensions can be installed using the requirements file i.e.,
-`pip install -r requirements.txt`. Additionally, building the developer
-documentation requires [Graphviz](https://graphviz.org/) for generating
-flowcharts.
+necessary extensions can be installed using the requirements file i.e., `pip
+install -r requirements.txt`.
```

Comment:
Graphviz needs to be installed separately (see [this comment](https://github.com/LLNL/sundials/pull/444/files#r1545523642))